config/peerpods/podvm/Dockerfile.podvm-builder

```diff
@@ -26,6 +26,9 @@ RUN mkdir -p /scripts
 
 ADD lib.sh libvirt-podvm-image-handler.sh aws-podvm-image-handler.sh azure-podvm-image-handler.sh /scripts/
 
+RUN mkdir -p /scripts/bootc
```

Comment:
nit, typo in the commit message
s/precarted/pre-created 
or s/precarted/pre-built